lib/services/screen-builder-service.ts

```diff
@@ -5,18 +5,24 @@ import * as path from "path";
 import * as util from "util";
 
 export class ScreenBuilderService implements IScreenBuilderService {
+	private static SCREEN_BUILDER_SPECIFIC_FILES = [".yo-rc.json", ".app.json", "app.js"];
+	private static UPGRADE_ERROR_MESSAGE = 'Your app has been build with an obsolete version of Screen Builder. Please, migrate it to latest version and retry.';
```

Comment:
Some time later (after a few discussions and investigations), it seems that we show this message in all cases. We need to update the message to say: "Your app has been created with an obsolete version of Screen Builder. You need to upgrade your to be able to run Screen Builder-related commands."